test/index.js

```diff
@@ -192,75 +192,77 @@ describe('PLACEHOLDER', () => {
     expect(stylis(`@import url('http://example.com');`)).to.equal(`@import url('http://example.com');`)
   })
 
-  // TODO: declarations in at-rule are not grouped in a rule
-  // test('@supports', () => {
-  //  expect(stylis(`
-  //      @supports (display:block) {
-  //        color:red;
-  //        h1 {
-  //          color:red;
-  //          h2 {
-  //            color:blue;
-  //          }
-  //        }
-  //        display:none;
-  //      }
-  //      @supports (appearance: none) {
-  //        color:red;
-  //      }
-  //      @supports (backdrop-filter: blur(10px)) {
-  //        backdrop-filter: blur(10px);
-  //      }
-  //    `)).to.equal([
-  //      '@supports (display:block){.user{color:red;display:none;}.user h1{color:red;}.user h1 h2{color:blue;}}',
-  //      '@supports (appearance:none){.user{color:red;}}',
-  //      '@supports (backdrop-filter:blur(10px)){.user{backdrop-filter:blur(10px);}}'
-  //    ].join(''))
-  // })
+  test('@supports', () => {
+    expect(
+      stylis(`
+        @supports (display:block) {
+          color:red;
+          h1 {
+            color:red;
+            h2 {
+              color:blue;
+            }
+          }
+          display:none;
+        }
+        @supports (appearance: none) {
+          color:red;
+        }
+        @supports (backdrop-filter: blur(10px)) {
+          backdrop-filter: blur(10px);
+        }
+     `)
+    ).to.equal([
+      '@supports (display:block){.user{color:red;display:none;}.user h1{color:red;}.user h1 h2{color:blue;}}',
+      '@supports (appearance:none){.user{color:red;}}',
```

Comment:
this test still ain't passing completely, because produced output looks like this:
```diff
-@supports (appearance: none)
+@supports (appearance:none)
```
Should we aim to remove this whitespace? If yes - I would prefer working on it in a separate PR though.

Reply:
No need, whitespace in () is treated as special because of whitespace in calc() having semantic meaning.

Reply:
right, this situation is slightly different though so we could special case it, not sure what exact goals of stylis are in regards to minification of the output etc

Reply:
anyway - I've added those whitespaces to the expected output and all tests are passing now 🚀

Reply:
I think this sums up the whitespace issue within calc() functions – https://stackoverflow.com/questions/34419813/why-must-a-or-be-surrounded-with-whitespace-from-within-the-calc-method is feasable to handle it. If you have any idea's you can take a stab, i'll also think about how that could be done, though not a priority right now.

Reply:
My main goal is to achieve correctness, so definitely not a priority for me either. I was just asking it as a general question. Thanks for the SO link!